OpenRA.Mods.Cnc/Traits/World/JumpjetActorLayer.cs

```diff
@@ -56,7 +56,7 @@ public JumpjetActorLayer(Actor self, JumpjetActorLayerInfo info)
 							continue;
 
 						neighbourCount++;
-						neighbourHeight += map.Height[neighbour];
+						neighbourHeight += ((IMapElevation)map).Height[neighbour];
```

Comment:
Check that map implements the interface first and throw if it doesn't (see tileset code).